x-pack/plugins/siem/public/components/stat_items/areachart.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+// @ts-ignore
```

Comment:
I was able to remove this `@ts-ignore` and no errors were shown.

Reply:
Thanks so much for giving it a try!